registry/storage/driver/middleware/cloudfront/middleware.go

```diff
@@ -124,6 +198,17 @@ func (lh *cloudFrontStorageMiddleware) URLFor(ctx context.Context, path string,
 		return lh.StorageDriver.URLFor(ctx, path, options)
 	}
 
+	addr, err := parseIPFromRequest(ctx)
```

Comment:
As mentioned above about making `awsIPs` nillable, this block should be skipped when accepted regions is not provided

Reply:
As replied above, we still need to check if the IP is an AWS, no matter whether accpetedRegion is provided or not